### DISCUSSION TITLE
How to build .net 8.0 sdk wihout using sdk?

### DISCUSSION THREAD

**Discussion by DancingCorpse** [dotnet/source-build#4564] [Q&A]
I want to build the .net 8.0 sdk from source on my machine, so I cloned the repository "dotnet/dotnet" and checkout the branch "release/8.0.1xx", but when running the command "./prep.sh", but it always tries to download the already built .net sdk and use this sdk to build a new sdk.

Is there a way to build the .net 8.0 sdk from source but without using the sdk?

**Comment by mthalman** [comment]
> Is there a way to build the .net 8.0 sdk from source but without using the sdk?

No, .NET requires a previous version of the .NET SDK to build. This is described in the [bootstrap guidelines doc](https://github.com/dotnet/source-build/blob/main/Documentation/bootstrapping-guidelines.md).
